packages/unplugin-typia/src/core/typia.ts

```diff
@@ -118,17 +118,28 @@ async function getProgramAndSource(
 	if (aliases != null && aliases.length > 0) {
 		host.resolveModuleNameLiterals = (moduleLiterals, containingFile, _, options) => {
 			return moduleLiterals.map((lit) => {
-				let module = ts.resolveModuleName(lit.text, containingFile, options, host, host.getModuleResolutionCache?.());
+				const module = ts.resolveModuleName(lit.text, containingFile, options, host, host.getModuleResolutionCache?.());
+
+				/* if module is resolved, return it */
+				if (module.resolvedModule != null) {
+					return module;
+				}
+
+				/* find matching alias */
 				const alias = findMatchingAlias(lit.text, aliases);
-				if (module.resolvedModule == null && alias != null) {
-					module = ts.resolveModuleName(
-						resolve(lit.text.replace(alias.find, alias.replacement)),
-						containingFile,
-						options,
-						host,
-					);
+
+				/* if no matching alias, return module */
+				if (alias == null) {
+					return module;
 				}
-				return module;
+
+				/* resolve alias */
+				return ts.resolveModuleName(
+					resolve(lit.text.replace(alias.find, alias.replacement)),
+					containingFile,
+					options,
+					host,
```

Comment:
We should add `host.getModuleResolutionCache?.()` here as well, I missed it previously

Reply:
OK I'll add it!

Reply:
I added at [5e4ea19](https://github.com/ryoppippi/unplugin-typia/pull/288/commits/5e4ea1986f901b9b23537e88db7f8acb894f0903)